charts/sn-platform/values.yaml

```diff
@@ -28,7 +28,7 @@ initialize: false
 # NOTE: Make the default namespace as empty. So it will fallback to use the namespace used for installing the helm
 #       chart. Helm does not position it self as a namespace manager, as namespaces in kubernetes are considered as
 #       a higher control structure that is not part of the application.
-namespace: ""
+namespace: "zxc"
```

Comment:
need to remove it

Reply:
oops, had to add namespace for testing, will remove it.